mmeval/metrics/end_point_error.py

```diff
@@ -150,12 +150,12 @@ def end_point_error_map(
         return epe.mean().cpu().numpy(), int(val.sum())
 
     @dispatch  # noqa: F811
-    def end_point_error_map(
+    def end_point_error_map(  # noqa: F811
         self,
         prediction: 'oneflow.Tensor',
         label: 'oneflow.Tensor',
-        valid_mask: Optional['oneflow.Tensor'] = None
-    ) -> Tuple[np.ndarray, int]:
+        valid_mask: Optional['oneflow.Tensor'] = None) -> Tuple[np.ndarray,
+                                                                int]:
```

Comment:
This seems a not necessary changes. It may be related to the problem of pre-commit~